nat-lab/tests/conftest.py

```diff
@@ -363,3 +364,21 @@ def copy_file_from_container(container_name, src_path, dst_path):
         )
     except subprocess.CalledProcessError:
         print(f"Error copying log file {src_path} from {container_name} to {dst_path}")
+
+
+async def collect_mac_diagnostic_reports():
+    if not (
+        "GITLAB_CI" in os.environ or "NATLAB_COLLECT_MAC_DIAGNOSTIC_LOGS" in os.environ
+    ):
+        return
+    print("Collect mac diagnostic reports")
+    try:
+        async with mac_vm_util.new_connection(reenable_nat=False) as connection:
```

Comment:
while running this locally, if mac VM is not present, I'll get an exception in here. Maybe just add `if CI:` check and then raise an exception, otherwise just ignore and proceed.

Reply:
Done